awesome_owl/static/src/todolist/todoitem.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import { Component } from "@odoo/owl";
+
+export class TodoItem extends Component {
+    static template = "awesome_owl.todoitem";
+    static props = {
+        todo: {
+            id: { type: Number },
+            description: { type: String },
+            isCompleted: { type: Boolean },
+        },
+        removeTodo: { type: Function },
+    };
+
+    toggleState() {
+        this.props.todo.isCompleted = !this.props.todo.isCompleted
```

Comment:
It's not really a good way to mutate a props. It's better to pass a function as a props and call it